CoreFoundation/Base.subproj/CFBase.h

```diff
@@ -138,6 +138,10 @@
             #define CF_EXPORT extern __declspec(dllimport)
         #endif
     #endif
+#elif TARGET_OS_LINUX || TARGET_OS_BSD || TARGET_OS_ANDROID
+
+#define CF_EXPORT __attribute__((__visibility__("default")))
```

Comment:
Why don't we just define `CF_EXPORT` to include `extern`?

Reply:
Because the definition should not contain extern.  `CF_EXPORT` can be used on both the declaration and the definition.

Reply:
We define it to `extern` for our other platforms.

Reply:
Hmm, actually, now that I think about it, we should be able to use the visibility attribute on Darwin too and unify the codepaths.